src/docs/patterns/visualizations.mdx

```diff
@@ -48,9 +48,9 @@ Charts and graphs come in [many different forms](https://material.io/design/comm
     url={`https://material.io/design/communication/data-visualization.html`}
 />
 
-{/*
+/*
```

Comment:
not sure if this comment syntax works / plays nicely with prettier when using v2 of mdx